.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -75,8 +75,9 @@ jobs:
             -i ${{ matrix.container_img }} \
             bash -c "git config --global --add safe.directory /io && \
             cd /io && \
-            python3.10 -m pip install ninja && \
-            ln -s /opt/python/cp310-cp10/bin/ninja /usr/bin/ninja && \
+            echo $(which python3.10) && \
```

Comment:
Do we still need to print this and verbose ninja or was it for debugging purposes?  :thinking: